bins/tts/preprocess.py

```diff
@@ -228,7 +226,7 @@ def preprocess(cfg, args):
         print("Extracting content features for {}...".format(dataset))
```

Comment:
It seems that line225-227 is unrelated to TTS. You can have a double check and see whether to delete them.

Reply:
It can be kept for potential support in the furture.